src/module/functions/mod.rs

```diff
@@ -182,6 +182,20 @@ impl ModuleFunctions {
         &mut self.arena[id]
     }
 
+    /// Get a function ID by its name.
+    ///
+    /// Note that function names are *not* guaranteed to be unique. This will
+    /// return the first function in the module with the given name.
```

Comment:
It might be good to mention here that the filtering is done by the name subsection, not by the export name